Added toggle visibility support using plain JavaScript

JA-EWW opened the pull request.

JA-EWW committed:
Added toggle visibility support using plain JavaScript

solth reviewed (changes requested):
Looks good, but the JS file seems to be missing. I would also suggest to remove the "TODO" lines mentioning the replacement for jd:hideableController and jd:hideableArea tags since the new JS functions accomplish exactly that.

JA-EWW committed:
Fixed the missing files, added correct relative path to toggle.js and toggle.js.css script in head.xhtml

and removed TODOs in several files related with the <jd:hideableController> and <jd:hideableArea>
